src/lib/registration/ui/addon_selection_base_dialog.rb

```diff
@@ -76,12 +86,12 @@ def addon_selected?(_addon)
       # @return [Yast::Term] the main UI dialog term
       def content
         VBox(
-          VStretch(),
           Left(Heading(heading)),
+          Left(CheckBox(Id(:filter_beta), Opt(:notify),
```

Comment:
Um, what about hiding the `Filter Out Beta Versions` checkbox when there is no Beta add-on available? To me it looks a bit confusing when you change the check box state and the list does not change... 

Just tried the code with SLE12-GA (using the `FAKE_BASE_PRODUCT=1` hack) and I found the behavior a bit strange, the first idea was that the filtering was broken...